config/prod/studies/mobile-toolbox-pgdvpj.yaml

```diff
@@ -0,0 +1,30 @@
+template:
```

Comment:
Look good but do we need to add to the crontab file? See the most recent merged stack addition.

Reply:
Oops, didn't `git add` it